Svc/CmdDispatcher/CommandDispatcherImpl.cpp

```diff
@@ -188,4 +188,20 @@ void CommandDispatcherImpl::pingIn_handler(FwIndexType portNum, U32 key) {
     this->pingOut_out(0, key);
 }
 
+void CommandDispatcherImpl::seqCmdBuff_overflowHook(FwIndexType portNum, Fw::ComBuffer& data, U32 context){
+    // Extract command opcode
+    Fw::CmdPacket cmdPkt;
+    Fw::SerializeStatus stat = cmdPkt.deserialize(data);
+    U32 opcode = 0xDEADBEEF; // Note: 0xDEADBEEF = unable to extract cmd opcode       
```

Comment:
Probably should use `FwOpcodeType` instead of U32

Reply:
Can we use 0x00000000 as the invalid opcode?  EventManager already uses 0x00000000 as a reserved opcode, and so it would be natural to "reserve" the same id.